### PR TITLE
Consuming api

### PR DESCRIPTION
**Pull request by initseis**
In this branch I:

- Consumed the leaderboard API
- Used async and await to fetch the leaderboard API
- Created my new game and got my unique code
- Built my project with Webpack

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Kindly change regular functions to arrow functions.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -2,17 +2,46 @@ import './style.css';
...
+    .then((response) => response.json())
+    .then((json) => {
+      ul.innerHTML = '';
+      json.result.forEach((element) => {
+        const li = document.createElement('li');
+        const p = document.createElement('p');
+        p.innerHTML = `${element.user}: ${element.score}`;
+        li.appendChild(p);
+        ul.appendChild(li);
+      });
```
Comment (mikenath223) [inline src/index.js]: - [ ] Kindly change this and other regular functions to es6 arrow functions. If you open [this link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) it takes you to point 4 in the project requirements on Javascript best practices. There it encourages the use of ES6. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look like modern code. Take a minute and also go through [this article on why ES6 arrow functions are so awesome](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html).

Comment (initseis) [reply src/index.js]: I forgot it sorry. I already made the change.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by initseis** [comment]
Thanks, @mikenath223.

**Pull request merged**